templates/base/package.json

```diff
@@ -7,8 +7,9 @@
     "build": "next build",
     "start": "next start dev",
     "export": "next export",
-    "storybook:run": "cd storybook && yarn install && yarn storybook",
-    "storybook:build": "cd storybook && yarn install && yarn build-storybook"
+    "storybook:install": "cd storybook && yarn install",
```

Comment:
Well I wrote the same before, but reverted as had to switch to directory again (-_-)